### ISSUE TITLE
Transparent mode

### ISSUE THREAD

**Issue by hesan-aminiloo** [kkoomen/pointless#4]
Hey! I think it's a good idea to support a transparent mode where you can draw on a canvas that overlays the active screen.

This way we can highlight or explain something that we're seeing on the browser or somewhere else. I can say the problem or challenge might be that on macOS when an app goes to fullscreen mode, no other apps can be opened on top of that.

**Comment by kkoomen** [comment]
Hi,

interesting idea! I'll definitely look into it. I did saw someone else implement this with tauri, so it shouldn't be too much of a hassle to implement. I think MacOS users should just maximize the window, rather than go to fullscreen. I use MacOS myself and as you said, no other windows can go on top or below once going fullscreen.

**Comment by hesan-aminiloo** [comment]
Yeah great, I tried to implement something like this with Electron a while ago, maybe I forked this repo and tried to implement it myself to see how far I can go 😊
Thanks for this great app.

**Comment by kkoomen** [comment]
@hesan-aminiloo I just implemented the feature in basic form at the new `feature/transparency` branch. I do have to change the way the eraser works. The eraser is basically a brush where you draw with the background color. I have to think about how to do this when there's transparency involved. I couldn't use masks before, because I had to rewrite the structure, but I already rewrote it, so I think I can use masks, although they're quite annoying to work with.
